### ISSUE TITLE
Don't log when Akka is started

### ISSUE THREAD

**Issue by jroper** [playframework/playframework#4554]
There's no need to log when Akka is started at info level, it just adds noise:

```
--- (Running the application, auto-reloading is enabled) ---

[info] p.a.l.c.ActorSystemProvider - Starting application default Akka system: application
[info] p.c.s.NettyServer$ - Listening for HTTP on /0:0:0:0:0:0:0:0:9000

(Server started, use Ctrl+D to stop and go back to the console...)

[info] Compiling 7 Scala sources and 1 Java source to /Users/jroper/tmp/play-scala/target/scala-2.11/classes...
[info] - play.api.libs.concurrent.ActorSystemProvider - Starting application default Akka system: application
[info] - play.api.Play$ - Application started (Dev)
```

**Comment by richdougherty** [comment]
I agree, I was thinking about this the other day.

**Comment by Mironor** [comment]
These logs are exceptionally annoying when using `withApplication` with `around` method in tests as it shows two log messages per spec.
Following minimalistic logback config will hide them (and any other INFO/DEBUG log) until needed:
```
<configuration>

    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
        <encoder>
            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
        </encoder>
    </appender>

    <root level="WARN">
        <appender-ref ref="STDOUT" />
    </root>

</configuration>
```

**Comment by marcospereira** [comment]
This was fixed in #4743.
